src/shims/manual/mscorlib.csproj

```diff
@@ -8,9 +8,13 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'" />
   <ItemGroup>
     <Compile Include="mscorlib.forwards.cs" />
+    <ExcludedReferences Include="$(_RuntimePath)Microsoft.Diagnostics.Tracing.TraceEvent.dll;$(_RuntimePath)Microsoft.DiaSymReader.Native.*.dll" />
+
+    <!-- System.ValueTuple has a netstandard configuration so it is bin placed in the target runtime when generating facades for 2.0 and it conflicts with the already defined type in System.Private.CoreLib -->
```

Comment:
I thought you were going to fix this with a placeholder configuration in master?

Reply:
If we want to port this later to the release branch (since 2.2 is going to be branched of from 2.1 branch) that wouldn't work because of the placeholder configurations' bug that we have in the 2.1 release branch. However I could fix that in buildtools' release branch as well.

Reply:
*if* we port it, we use this workaround at that point.  I don't want the output being wrong if we can avoid it.

Reply:
Ok, I will use the placeholder then.